<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-1DVQM3GP" failures="1" tests="1" name="Test" time="6.372" errors="0" timestamp="2024-04-11T15:45:04 IST">
  <testcase classname="testcases.NegativeScenarios" name="myTest" time="0.939">
    <failure type="java.lang.AssertionError" message="Error message for trying to login with Locked out user is not matching expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error message for trying to login with Locked out user is not matching expected [true] but found [false]
at testcases.NegativeScenarios.myTest(NegativeScenarios.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
</testsuite> <!-- Test -->
